src/Driver/LazyRabbitMqDriver.php

```diff
@@ -39,7 +39,7 @@ public function __construct(AMQPLazyConnection $connection, string $queue)
      */
     public function send(MessageInterface $message): bool
     {
-        $rabbitMessage = new AMQPMessage($this->serializer->serialize($message));
+        $rabbitMessage = new AMQPMessage($this->serializer->serialize($message), ['delivery_mode' => AMQPMessage::DELIVERY_MODE_PERSISTENT]);
```

Comment:
What about move this settings array to constructor of LazyRabbitMqDriver as third param? it could be default value but it will be more general. Also for other Rabbit driver.